include/imex/Conversion/GPUXToLLVM/GPUXToLLVMPass.h

```diff
@@ -0,0 +1,52 @@
+//===- GPUXToLLVMPass.h - GPUXToLLVM conversion  ---------------*- C++ -*-===//
+//
+// Copyright 2022 Intel Corporation
+// Part of the IMEX Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+///
+/// \file
+/// This file defines the GPUXToLLVM conversion,
+///
+//===----------------------------------------------------------------------===//
+
+#ifndef IMEX_GPUXTOLLVM_PASS_H_
+#define IMEX_GPUXTOLLVM_PASS_H_
+
+#include <memory>
+#include <mlir/IR/PatternMatch.h>
+#include <mlir/Transforms/DialectConversion.h>
```

Comment:
I don't think these includes are needed. And some forward declarations can also be removed.